llvm/lib/Target/RISCV/RISCVISelLowering.cpp

```diff
@@ -13447,7 +13447,6 @@ static SDValue expandMul(SDNode *N, SelectionDAG &DAG,
   case 21:
   case 25:
   case 27:
-  case 29:
```

Comment:
Why is it removed?

Reply:
It was a mistake to start with.  We don't actually have a TD pattern for 29.